Header.h

```diff
@@ -5,5 +5,60 @@
 // Ayisha Sameera,GET.
 // ------------------------------------------------------------------
 // Header.h
-// Program on branch main.
+// Program on branch A5.
+// Assignment 5 : Implementation of heap sort and binary search
 // ------------------------------------------------------------------------------------------------
+
+///<summary>
+/// Heap Sort is a comparison-based sorting algorithm that utilizes a binary heap data structure. 

```

Comment:
The description on the algorithm is only required for the dev that'll be reading/editing this. Please move this to the same file as the algo, and use the // type comments.